Test failure: `com.linecorp.armeria.server.InvalidPathWithDataTest.invalidPath()`

```
java.io.IOException: RST_STREAM received
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:964)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)
	at com.linecorp.armeria.server.InvalidPathWithDataTest.invalidPath(InvalidPathWithDataTest.java:76)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.io.IOException: RST_STREAM received
	at java.net.http/jdk.internal.net.http.Stream.incoming_reset(Stream.java:586)
	at java.net.http/jdk.internal.net.http.Stream.otherFrame(Stream.java:501)
	at java.net.http/jdk.internal.net.http.Stream.incoming(Stream.java:494)
	at java.net.http/jdk.internal.net.http.Http2Connection.processFrame(Http2Connection.java:936)
	at java.net.http/jdk.internal.net.http.frame.FramesDecoder.decode(FramesDecoder.java:155)
	at java.net.http/jdk.internal.net.http.Http2Connection$FramesController.processReceivedData(Http2Connection.java:307)
	at java.net.http/jdk.internal.net.http.Http2Connection.asyncReceive(Http2Connection.java:778)
	at java.net.http/jdk.internal.net.http.Http2Connection$Http2TubeSubscriber.processQueue(Http2Connection.java:1594)
	at java.net.http/jdk.internal.net.http.common.SequentialScheduler$LockingRestartableTask.run(SequentialScheduler.java:182)
	at java.net.http/jdk.internal.net.http.common.SequentialScheduler$CompleteRestartableTask.run(SequentialScheduler.java:149)
	at java.net.http/jdk.internal.net.http.common.SequentialScheduler$SchedulableTask.run(SequentialScheduler.java:207)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

```

--- a/core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java
+++ b/core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java
@@ -29,6 +29,7 @@ import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicInteger;
 
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.api.Test;
 
 import com.linecorp.armeria.common.HttpData;
@@ -152,6 +153,7 @@ class ByteStreamMessageOutputStreamTest {
 
     @Test
     void writeAfterStreamClosed() throws InterruptedException {
+        final CountDownLatch abortWait = new CountDownLatch(1);
         final CountDownLatch wait = new CountDownLatch(1);
         final CountDownLatch end = new CountDownLatch(1);
         final ByteStreamMessage byteStreamMessage = StreamMessage.fromOutputStream(os -> {
@@ -160,6 +162,7 @@ class ByteStreamMessageOutputStreamTest {
                     if (i < 2) {
                         os.write(i);
                     } else {
+                        abortWait.countDown();
                         wait.await();
                         assertThatThrownBy(() -> os.write(0))
                                 .isInstanceOf(IOException.class)
@@ -175,6 +178,13 @@ class ByteStreamMessageOutputStreamTest {
 
         StepVerifier.create(byteStreamMessage, 2)
                     .expectNext(httpData(0), httpData(1))
+                    .then(() -> {
+                        try {
+                            abortWait.await();
+                        } catch (InterruptedException e) {
+                            throw new RuntimeException(e);
+                        }
+                    })
                     .then(byteStreamMessage::abort)
                     .then(() -> {
                         // Wait for the abortion to be completed.

--- a/core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java
+++ b/core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java
@@ -29,7 +29,6 @@ import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicInteger;
 
 import org.hamcrest.Matchers;
-import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.api.Test;
 
 import com.linecorp.armeria.common.HttpData;

--- a/core/src/test/java12/com/linecorp/armeria/server/InvalidPathWithDataTest.java
+++ b/core/src/test/java12/com/linecorp/armeria/server/InvalidPathWithDataTest.java
@@ -27,9 +27,11 @@ import org.junit.jupiter.api.extension.RegisterExtension;
 
 import com.linecorp.armeria.common.HttpResponse;
 import com.linecorp.armeria.common.RequestTarget;
+import com.linecorp.armeria.internal.testing.FlakyTest;
 import com.linecorp.armeria.server.logging.LoggingService;
 import com.linecorp.armeria.testing.junit5.server.ServerExtension;
 
+@FlakyTest
 class InvalidPathWithDataTest {
 
     @RegisterExtension

